tests/resource/test_resource_setters.py

```diff
@@ -18,10 +18,278 @@ class TestResourceSetters(unittest.TestCase):
+        self.assertEqual(
+            result,
+            TestResourceConstants.TEST_RESOURCE_OVERWRITE_AUDIT_BY_ACCESS_LEVEL_REQUEST_XML,
+        )
+
+    def test_resource_admin_build_overwrite_audit_by_access_level_none_request(self):
+        result = self.resource_admin.overwrite_audit_by_access_level(
+            "TESTING", "ELIJTEST"
+        )
+        self.assertEqual(
+            result,
+            TestResourceConstants.TEST_RESOURCE_OVERWRITE_AUDIT_BY_ACCESS_LEVEL_NONE_REQUEST_XML,
+        )
+
+    def test_resource_admin_build_overwrite_audit_by_access_level_multiple_request(
+        self,
+    ):
+        result = self.resource_admin.overwrite_audit_by_access_level(
+            "TESTING", "ELIJTEST", alter="success", control="all"
+        )
+        self.assertEqual(
+            result,
+            TestResourceConstants.TEST_RESOURCE_OVERWRITE_AUDIT_BY_ACCESS_LEVEL_MULT_REQUEST_XML,
+        )
+
+    def test_resource_admin_build_overwrite_audit_by_access_level_all_request(self):
+        result = self.resource_admin.overwrite_audit_by_access_level(
+            "TESTING",
+            "ELIJTEST",
+            alter="success",
+            control="all",
+            update="success",
+            read="failure",
+        )
+        self.assertEqual(
+            result,
+            TestResourceConstants.TEST_RESOURCE_OVERWRITE_AUDIT_BY_ACCESS_LEVEL_ALL_REQUEST_XML,
+        )
+
+    def test_resource_admin_build_overwrite_audit_by_attempt_request(self):
+        result = self.resource_admin.overwrite_audit_by_attempt(
+            "TESTING", "ELIJTEST", failure="control"
+        )
+        self.assertEqual(
+            result,
+            TestResourceConstants.TEST_RESOURCE_OVERWRITE_AUDIT_BY_ATTEMPT_REQUEST_XML,
+        )
+
+    def test_resource_admin_build_overwrite_audit_by_attempt_none_request(self):
+        result = self.resource_admin.overwrite_audit_by_attempt("TESTING", "ELIJTEST")
+        self.assertEqual(
+            result,
+            TestResourceConstants.TEST_RESOURCE_OVERWRITE_AUDIT_BY_ATTEMPT_NONE_REQUEST_XML,
+        )
+
+    def test_resource_admin_build_overwrite_audit_by_attempt_multiple_request(self):
+        result = self.resource_admin.overwrite_audit_by_attempt(
+            "TESTING", "ELIJTEST", success="alter", all="read"
+        )
+        self.assertEqual(
+            result,
+            TestResourceConstants.TEST_RESOURCE_OVERWRITE_AUDIT_BY_ATTEMPT_MULT_REQUEST_XML,
+        )
+
+    def test_resource_admin_build_overwrite_audit_by_attempt_all_request(self):
+        result = self.resource_admin.overwrite_audit_by_attempt(
+            "TESTING", "ELIJTEST", success="alter", all="read", failure="update"
+        )
+        self.assertEqual(
+            result,
+            TestResourceConstants.TEST_RESOURCE_OVERWRITE_AUDIT_BY_ATTEMPT_ALL_REQUEST_XML,
+        )
+
+    def test_resource_admin_build_overwrite_audit_by_attempt_value_error(self):
+        bad_val = "problem"
```

Comment:
`bad_success = "problem"` instead of `bad_val = "problem"`.

Reply:
very fair point. changed.